src/components/bread-crumb-nav/bread-crumb-nav.js

```diff
@@ -1,42 +1,50 @@
 import { Breadcrumb } from '@fluentui/react/lib/Breadcrumb';
+import { PATHS, ROUTE_NAME_BY_PATH } from 'common/constants';
+import { getSplitPaths } from 'common/functions';
+import { useConversionStore, useUserStore } from 'common/store';
+import { useEffect } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useNavigate, useLocation } from 'react-router-dom';
-
+import { useLocation, useNavigate } from 'react-router-dom';
+import { shallow } from 'zustand/shallow';
 import { breadcrumbStyle } from './bread-crumb-nav.style';
-import { getSplitPaths } from 'common/functions';
-import { PATHS, ROUTE_NAME_BY_PATH } from 'common/constants';
-import { useConversionStore } from 'common/store';
 
-const routesConfirmationRequiredBeforeLeaving = [PATHS.CONVERSION];
+const routesReset = [PATHS.CONVERSION];
+
+const userStoreSelector = s => [s.subscriptionKey];
 
 const BreadCrumbNav = () => {
   const { t } = useTranslation();
   const navigate = useNavigate();
-  const { pathname } = useLocation();
+  const { pathname: currentPath } = useLocation();
+  const [subKey] = useUserStore(userStoreSelector, shallow);
 
-  const splitPaths = getSplitPaths(pathname);
+  const splitPaths = getSplitPaths(currentPath);
   const itemsWithHeading = [];
 
-  splitPaths.forEach(path => {
-    if (ROUTE_NAME_BY_PATH.hasOwnProperty(path)) {
+  useEffect(() => {
+    if (!subKey) {
+      navigate(PATHS.INDEX, { replace: true });
```

Comment:
similar code exists here https://github.com/Azure/azure-maps-creator-onboarding-tool/blob/main/src/components/route/route.js#L31, i guess one of these can be removed

Reply:
Fixed in https://github.com/Azure/azure-maps-creator-onboarding-tool/pull/104